src/config.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+return [
+    /**
```

Comment:
have a look here https://github.com/EscolaLMS/settings/issues/7

can you wrap those settings in `public` key ?